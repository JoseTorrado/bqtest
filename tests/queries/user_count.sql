SELECT country, COUNT(*) as user_count
FROM ${TABLE}
GROUP BY country
ORDER BY country
