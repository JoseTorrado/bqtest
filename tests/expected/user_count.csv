country,user_count
Canada,2
UK,1
USA,2
